docs/hyperdatasets/webapp/webapp_dataviews.md

```diff
@@ -2,11 +2,11 @@
 title: The Dataviews Table
 ---
 
-[Dataviews](../dataviews.md) appear in the same Project as the experiment that stored the Dataview in the ClearML Enterprise platform, 
-as well as the **DATAVIEWS** tab in the **All Projects** page.
-
 The **Dataviews table** is a [customizable](#customizing-the-dataviews-table) list of Dataviews associated with a project.
-Use it to view, create, and edit Dataviews in the info panel. 
+Use it to view and create Dataviews, and access their info panels. 
+
+The table lists independent Dataview objects. To see Dataviews logged by a task, go
```

Comment:
@pollfly Not necessarily "logged by a task". More like a task's specific dataviews.